HandsOnTutorial/1-hello-world.fsx

```diff
@@ -1,15 +1,19 @@
 ﻿(*** hide ***)
-#load "credentials.fsx"
-
+#load "Thespian.fsx"
+#load "Azure.fsx"
 
 open System
 open System.IO
 open MBrace.Core
-open MBrace.Azure
 open MBrace.Flow
 
+// Initialize client object to an MBrace cluster:
+let cluster = 
+//    getAzureClient() // comment out to use an MBrace.Azure cluster; don't forget to set the proper connection strings in Azure.fsx
+    initThespianCluster(4) // use a local cluster based on MBrace.Thespian; configuration can be adjusted using Thespian.fsx
```

Comment:
This looks awkward.  Too many names for the first line of MBrace code :) "Azure", "Client", "Thespian", "init", "Cluster".  

If you switch to making the choice depdnent on which .fsx is used, then you can just stick to ``let cluster = GetCluster(config)``  where the script defines ``GetCluster``